openrtb3/parse_time_test.go

```diff
@@ -0,0 +1,61 @@
+package openrtb3_test
+
+import (
+	"time"
+
+	. "github.com/mxmCherry/openrtb/openrtb3"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("ParseTime", func() {
```

Comment:
Timezone parsing works well on my PC, but fails in travis.
Probably, better keep this lib dumb and just remove `ParseTime` helper entirely.

Reply:
Luckily, this is gone from FINAL spec.